ROADMAP.md

```diff
@@ -1 +1,39 @@
+# Road Map
 
+## What
+We want to build an application/tool that can import a dataset, store the relevant information in a database, then dispay the data to the user. We will accomplish this in 3 milestones:  
+
+1. Set up repository, dev environments, and initial files. Project can build, run and connect to the database. (complete)
+2. Able to import data into the database on server and get data with a basic interface. (~2 weeks)
+3. Able to perform transformations/calculations on the data and display it with a simple web-page. (~3 weeks)
+4. Make the web-page interactive. (~2 weeks)
```

Comment:
I'd be more specific with this too. What does interactive mean? Allow the user to be able to transform the data themselves? Allow the user to request transformations on the data to be performed themselves, and display this transformation?

Reply:
I have added more detail in the latest [commit](https://github.com/menghaoyu2002/CSC302-TeamRocket/pull/14/commits/2e781e25ec900fa2fbc6847796c12bebb57ad53a)